meta-resin-common/recipes-support/resin-device-register/resin-device-register/resin-device-register

```diff
@@ -48,21 +48,21 @@ while true; do
     then
         # The `apiKey` default is mainly so that we don't hit issues where we've already registered
         # and then everything goes out of line due to an empty apiKey
-        read api_key user_id application_id device_type uuid device_api_key registered_at <<<$(jq -r '.apiKey // .deviceApiKey,.userId,.applicationId,.deviceType,.uuid,.deviceApiKey,.registered_at // empty' $CONFIG_PATH)
+        _device_api_key=${PROVISIONING_API_KEY:-DEVICE_API_KEY}
```

Comment:
Yeah you're right, previously the default was just to stop misalignment issues, but now that resin-vars exposes the env vars for us there's no need to have that hack